calour/ms1_experiment.py

```diff
@@ -196,3 +161,129 @@ def get_bad_features(self, mz_tolerance=0.001, rt_tolerance=2, corr_thresh=0.8,
                 keep_features.extend(ckeep)
                 features.drop(index=ckeep, inplace=True)
         return self.filter_ids(keep_features, negate=negate, inplace=inplace)
+
+    def merge_similar_features(self, mz_tolerance=0.001, rt_tolerance=0.5):
+        '''Merge metabolites with similar mz/rt to a single metabolite
+
+        metabolites are initially sorted by frequency and a greefy clustering algorithm (starting from the highest freq.) is used to join together
```

Comment:
1. pls write proper english sentences (eg use capital letter).

2. greefy -> greedy?

3. does this function really do clustering? maybe I missed it but I don't see it.

4. the description is still not clear.  You sum up the features with similar mz and rt? pls be more specific than `join together`.

Reply:
1,2,4 fixed
3. It is greedy clustering, similar to open-reference OTU picking (the highest freq. feature is the center, add all metabolites close enough to the center...)